cmd/podman/volume_prune.go

```diff
@@ -16,12 +16,9 @@ import (
 
 var (
 	volumePruneCommand     cliconfig.VolumePruneValues
-	volumePruneDescription = `
-podman volume prune
+	volumePruneDescription = `Command prompts for confirmation if not using the --force flag.
```

Comment:
No longer says what this command does ("Remove all unused volumes"). Was that intentional?

Reply:
Fixed